ci-automation/vendor-testing/qemu.sh

```diff
@@ -49,6 +49,7 @@ kola run \
     --qemu-image="${QEMU_IMAGE_NAME}" \
     --tapfile="${CIA_TAPFILE}" \
     --torcx-manifest="${CIA_TORCX_MANIFEST}" \
+    --qemu-skip-mangle \
```

Comment:
For backporting it would be great to have this change in a separate commit - we try to align `ci-automation/` for the branches but not the `.github` folder.